src/noit_metric.h

```diff
@@ -77,6 +79,7 @@ typedef enum {
 } noit_message_type;
 
 #define NOIT_TAG_MAX_COMPONENT_LEN 127
+#define NOIT_TAG_MAX_PAIR_LEN 256
```

Comment:
This might be too short.. Also, I would allow for shorter categories and longer values.